docs/internal/ci.md

```diff
@@ -0,0 +1,61 @@
+# E2E tests
+
+We use GitHub Actions to run an end-to-end tests to verify all PRs. The GitHub Actions used are a combination of `aws-actions/configure-aws-credentials` and `hashicorp/setup-terraform@v1`. See the complete action definition [here](../../.github/workflows/e2e-test.yml).
+
+## Setup
+
+1. Use the following CloudFormation template to setup a new IAM role.
+
+```yaml
```

Comment:
Nice to have this as a TF resource

Reply:
Hi @vara-bonthu, I spoke to @kcoleman731 and since this is meant for our internal usage we do not need a terraform example here.